components/BB.js

```diff
@@ -298,21 +283,9 @@ export class BB extends Component {
                     </FormControl>
                   </Grid>
 
-                  <Grid item xs={8} className={classnames(classes.collapse)}>
```

Comment:
I think there's a corresponding grid item that should have `xs={4}` changed to `xs={12}`